code/modules/surgery/organs/lungs.dm

```diff
@@ -21,7 +21,7 @@
 	food_reagents = list(/datum/reagent/consumable/nutriment = 5, /datum/reagent/medicine/salbutamol = 5)
 
 	//Breath damage
-	//These numbers are roughly equivilant to molar count at room temperature, they scale up and down with temp tho
+	//These numbers are equivilant to pressure taken up by that type of gas in the main mix
```

Comment:
```suggestion
	//These numbers are equivalent to pressure taken up by that type of gas in the main mix
```

Reply:
What main mix though, distro?

Reply:
I wrote this last night in a stupor, lemme redo it again